src/utils.js

```diff
@@ -64,3 +64,26 @@ export function getPackageTypeAsString(numericPackageType) {
   }
   throw new Error(`Invalid package type constant "${numericPackageType}"`);
 }
+
+/*
+ * Looks throuh all exported functions and returns only
+ * "public" functions that aren't prefixed with an _
+ *
+ * Used for ignoring private functions in rules files.
+ * Rules can have private functions we don't run; anything that
+ * starts with an "_" shouldn't be returned.
+ *
+ * This exists because we export private functions in rule files
+ * for testing.
+ */
+export function ignorePrivateFunctions(list) {
+  var filteredList = {};
+
+  for (let functionName in list) {
```

Comment:
Problem is, it’s actually an object (key/value) because they’re imported using `* as rules from ‘rules/html’`.

I’ll change the name of the argument to something that makes that clear and/or add a comment.

I actually used .filter at first then wondered why my tests failed ;-)